code/src/test/java/com/wenmq/leetcode/Solution190UnitTest.java

```diff
@@ -0,0 +1,40 @@
+package com.wenmq.leetcode;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class Solution190UnitTest {
+
+    private static final String TAG = "com.wenmq.leetcode.Solution190";
+
+    private Solution190 mTestSolution;
+
+
+    @Before
+    public void setUp() throws Exception {
+        mTestSolution = new Solution190();
+    }
+
+    @Test
+    public void testReverseBits_Case1() {
+        int n = Integer.valueOf(replaceMinus("00000010100101000001111010011100"), 2);
+        int expected = Integer.valueOf(replaceMinus("00111001011110000010100101000000"), 2);
+        Assert.assertEquals(expected, mTestSolution.reverseBits(n));
+    }
+
+    @Test
+    public void testReverseBits_Case2() {
+        int n = Integer.valueOf(replaceMinus("11111111111111111111111111111101"), 2);
+        int expected = Integer.valueOf(replaceMinus("10111111111111111111111111111111"), 2);
+        Assert.assertEquals(expected, mTestSolution.reverseBits(n));
+    }
+
+    private String replaceMinus(String a) {
```

Comment:
This method work for case 2. 
unsigned int may overflow when you use Integer.valueOf(String s, itn radix) because of the first bit(represent signed)

Reply:
when n < 0, use a '-' operator instead of leading '1' when use Integer.valueof(s,2)